BrainFlowPlugin/Source/BrainFlowPlugin/BrainFlowPlugin.Build.cs

```diff
@@ -35,9 +35,9 @@ public BrainFlowPlugin(ReadOnlyTargetRules Target) : base(Target)
 
             PublicAdditionalLibraries.Add(Path.Combine(PrecompiledFolder, "libBrainflow.a"));
 
-            PublicDelayLoadDLLs.Add("libBoardController.dylib");
-            PublicDelayLoadDLLs.Add("libDataHandler.dylib");
-            PublicDelayLoadDLLs.Add("libMLModule.dylib");
+            PublicDelayLoadDLLs.Add(Path.Combine(PrecompiledFolder, "libBoardController.dylib"));
```

Comment:
should it be the same for linux?

Reply:
I actually don't know without testing.

On Windows, I found the `PublicDelayLoadDLLs` need to be filename only and manually loaded in the module file to work, but they won't work with full path.

On Mac, I found the `PublicDelayLoadDLLs` will automatically get loaded properly if the full path is provided which I found easier and nice.

On Linux, I got no clue what will work, but I hope the full path will be working there too. I will make the adjustments